docs/src/css/custom.css

```diff
@@ -612,4 +612,13 @@ p strong {
   position: absolute;
   top: 0;
   left: 0;
-}
\ No newline at end of file
+}
+
+/*
+  The overview list components are enriched by swizzled DocCardList and DocSidebarItems wrappers.
+  Deprecated list item titles have the class zitadel-lifecycle-deprecated.
+  We strike them through, because this is well-known practice and reduces mental overhead for finding the right method to solve a task.
+ */
+.zitadel-lifecycle-deprecated {
+  text-decoration: line-through;
+}
```

Comment:
I'd suggest adding a "DEPRECATED" tag to each card item as well, for a bit more clarity. This is an easy way to do it with CSS..

```suggestion
.card:has(.zitadel-lifecycle-deprecated) {
  position: relative;
}

.card:has(.zitadel-lifecycle-deprecated)::after {
  content: "DEPRECATED";
  position: absolute;
  top: 10px;
  right: 10px;
  background-color: var(--ifm-color-danger-contrast-background);
  color: var(--ifm-color-danger-contrast-foreground);
  font-size: 12px;
  font-weight: 600;
  padding: 0.25rem 0.5rem;
  border-radius: 0.25rem;
  text-transform: uppercase;
  letter-spacing: 0.05em;
}

.zitadel-lifecycle-deprecated {
  text-decoration: line-through;
}
```

Reply:
Thanks @skewis6 I like this

![image](https://github.com/user-attachments/assets/c856a27a-c5ad-4f00-b02e-bab72d93b6c8)